docs/proposals/node-agent/agent.proto

```diff
@@ -0,0 +1,41 @@
+syntax = "proto3";
+
+package node_agent;
+
+enum WorkloadStatus {
+    WAITING = 0;
+    RUNNING = 1;
+    TERMINATED = 2;
+}
+
+message WorkloadRequest {
+    string id = 1;
+}
+
+enum WorkloadType {
+    CONTAINER = 0;
+}
+
+message WorkloadResource {
+    int32 cpu = 1;
+    int32 memory = 2;
+    int32 disk = 3;
+}
+
+message WorkloadDefinition {
+    string id = 1;
+    string name = 2;
+    WorkloadType type = 3;
+    string image = 4;
```

Comment:
Should the image be the FQN (Fully Qualified Name), such as `ghcr.io/example/nginx:latest`? If not, should we include the tag in another argument of the message?

Reply:
I'm open to any additions, what would be the benefits of having the tag separated ?